examples/nextjs-blog/pages/index.js

```diff
@@ -1,9 +1,174 @@
+              title={getImageAlt(post)}
+            />
+            <CardContent>
+              <p>{getDateString(date)}</p>
+              <p>{displayName}</p>
+              <Link route={`/blog/${slug}`}>
+                {/* href gets added to <a> by <Link> */}
+                <a>
+                  <h4>{title}</h4>
+                </a>
+              </Link>
+              <p>{excerpt}</p>
+            </CardContent>
+            <CardActions>
+              <Link route={`/blog/${slug}`}>
+                <a className="readPost">Read Article</a>
+              </Link>
+            </CardActions>
+          </Card>
+        </Grid>
+      )
+    })
+  }
+
+  function renderSuggestedPostSlider(home) {
+    console.log(home)
+    // return (
+    //   <Carousel
+    //     cellAlign="left"
+    //     heightMode="max"
+    //     wrapAround={false}
+    //     slidesToScroll={1}
+    //   >
+    //     {suggestedPosts.map(post => {
+    //       console.log(post)
+    //       const { title, url, slug } = post
+
+    //       return (
+    //         <div key={slug}>
+    //           <Link route={`/blog/${slug}`}>
+    //             <img src={url} alt={title} />
+    //           </Link>
+    //         </div>
+    //       )
+    //     })}
+    //   </Carousel>
+    // )
+  }
+
+  const { posts, home } = props
+
+  if (!posts) {
+    return <h4>No posts yet :(</h4>
+  }
+
   return (
-    <h1 style={{ textAlign: 'center' }}>
-      Welcome to the Flamelink + Next.js blog example!
-    </h1>
+    <div className={containerWide.className}>
+      {home.content}
+      {renderSuggestedPostSlider(home)}
+      <h2>Latest Posts</h2>
+      <Grid container justify="flex-start" spacing={32} className="pageSection">
+        {renderPostCards(posts)}
+      </Grid>
+      <style jsx>{`
+        :global(.media) {
+          height: 11.43rem;
+        }
+
+        :global(.card) {
+          height: 100%;
+        }
+
+        :global(.pageSection) {
+          margin-bottom: 4.28rem;
+        }
+      `}</style>
+      <style jsx>{containerWide}</style>
+    </div>
   )
 }
 
+Home.fields = [
+  'title',
+  'slug',
+  'author',
+  'image',
+  'date',
+  '_fl_meta_',
+  'suggested',
+]
+
+Home.populate = [
+  {
+    field: 'author',
+    fields: ['displayName'],
+  },
+]
+
+Home.propTypes = {
+  posts: PropTypes.array.isRequired,
+  home: PropTypes.shape({
+    content: PropTypes.string.isRequired,
+    suggestedPost: PropTypes.array.isRequired,
+  }).isRequired,
+}
+
+Home.getInitialProps = async () => {
+  const posts = Object.values(
+    (await app.content.get({
+      schemaKey: 'blogPost',
+      filters: [['status', '==', 'published']],
+      orderBy: {
+        field: 'date',
+        order: 'asc',
+      },
+      limit: 3,
+      populate: Home.populate,
+      fields: Home.fields,
+    })) || {}
+  )
+
+  const home = await app.content.get({
+    schemaKey: 'home',
+    // todo: jp - fix `populate` functionality for repeaters
```

Comment:
@jordanch found a problem with the `populate` functionality here, so I'll first have to fix it in the SDK before this can be completed. I updated the query to get the full home page content object. Will let you know once the `populate` function is working again.